package.json

```diff
@@ -87,6 +88,9 @@
     },
     "collectCoverageFrom": [
       "**/*.(t|j)s"
+    ],
+    "transformIgnorePatterns": [
+      "node_modules/(?!@dfx)"
     ]
```

Comment:
What is this for?

Reply:
we are importing `@dfx.swiss/react` into test files, this led to building issues which were resolved by adding this line